### PR TITLE
v1.24 Update to the Check if Dockershim Deprecation Affects You page

### PR DESCRIPTION
**Pull request by reylejano**
This PR has <strike>2</strike> 3 commits:
- update the tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecatation-affects-you page for the dockershim removal in v1.24
- updates any links to the renamed page
- add redirect

### CODE REVIEW

**Comment by nate-double-u** [comment]
Do we need to add a forward since we’re changing the URL?

**Review by sftim — CHANGES_REQUESTED** [review]
@reylejano, I'd also like a redirect so that the old URL still works. Other than that, great (and an important change to make).

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ weight: 20
 
 The `dockershim` component of Kubernetes allows to use Docker as a Kubernetes's
 {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
-Kubernetes' built-in `dockershim` component was deprecated in release v1.20.
+Kubernetes' built-in `dockershim` component was removed in release v1.24.
```
Comment (divya-mohan0209) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you.md:13]: Is the intention to publish this after the release? If yes, then the past tense makes sense. If not, we might have to reword it.

**Review by divya-mohan0209 — COMMENTED** [review]
Slight grammatical nit

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you.md:13]: We will publish this change (which targets _dev-1.24_) as part of the v1.24 release.

**Comment by reylejano** [comment]
> @reylejano, I'd also like a redirect so that the old URL still works. Other than that, great (and an important change to make).

@sftim, added redirect in accd25a

**Comment by bradtopol** [comment]
Assigning @divya-mohan0209 and @sftim since they are both already engaged on this
/assign @divya-mohan0209 
/assign @sftim

**Review by divya-mohan0209 — COMMENTED** [review]
This looks good to me. LGTM!

**Comment by divya-mohan0209** [comment]
Please feel free to remove the hold & merge the PR, @reylejano if there are no specific timelines around this.

**Pull request merged**